src/test/python/Makefile

```diff
@@ -0,0 +1,2 @@
+pybuild/vrs_pb2.py : vrs.proto
```

Comment:
File not needed when building with maven.

Reply:
Will remove. 👍